recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
   sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
-    - python.app  # [osx]
```

Comment:
Packages for 3.6 will come with Anaconda 4.3, to be released the last week of January.